test/files/run/patmat-behavior.check

```diff
@@ -136,6 +136,30 @@ patmat-behavior.scala:55: warning: match may not be exhaustive.
 It would fail on the following inputs: C00(), C01(_), C10(_), C11(_, _), C20(_, _), C21(_, _, _)
     def gc6[A](x: C[A]) = x match { case F00() => ??? ; case F10(x) => x ; case F20(x, y) => x ; case F01(xs @ _*) => xs.head ; case F11(x, ys @ _*) => x ; case F21(x, y, zs @ _*) => x }
                           ^
+patmat-behavior.scala:57: warning: match may not be exhaustive.
+It would fail on the following inputs: C00(), C01(_), C10(_), C11(_, _), C20(_, _), C21(_, _, _)
+    def gd1[A, B <: C[A]](x: B) = x match { case F00() => ??? ; case F10(x) => x ; case F20(x, y) => x ; case F01(xs @ _*) => xs.head ; case F11(x, ys @ _*) => x ; case F21(x, y, zs @ _*) => x }
```

Comment:
Notice the line above: this match is like the one above except instead of the scrutinee being `C[A]` it's `B` which is `<: C[A]`.  So warning in the same way SGTM.